test/spec/lib/clay-config.js

```diff
@@ -287,6 +289,20 @@ describe('ClayConfig', function() {
     });
   });
 
+  describe('.getItemsByGroup()', function() {
+    it('it returns the correct items', function() {
+      var config = fixtures.config([
+        {type: 'input', clayId: 0, group: 'group1'},
+        {type: 'text', clayId: 1, group: 'group2'},
+        {type: 'input', clayId: 2, group: 'group1'}
+      ]);
+      var clayConfig = fixtures.clayConfig(config);
+      assert.deepEqual(clayConfig.getItemsByGroup('group1')[0].config, config[0]);
+      assert.deepEqual(clayConfig.getItemsByGroup('group1')[1].config, config[2]);
+      assert.deepEqual(clayConfig.getItemsByGroup('group2')[0].config, config[1]);
```

Comment:
Rather do:

```js
var config = fixtures.config([
  {type: 'input', id: 'g1-0', group: 'group1'},
  {type: 'text', id: 'g2-0', group: 'group2'},
  {type: 'input', id: 'g1-1', group: 'group1'}
]);

var clayConfig = fixtures.clayConfig(config);
assert.deepEqual(clayConfig.getItemsByGroup('group1'), [
  clayConfig.getItemById('g1-0'),
  clayConfig.getItemById('g1-1')
]);
assert.deepEqual(clayConfig.getItemsByGroup('group2'), [
  clayConfig.getItemById('g1-1')
]);
```
This also ensure the result *only* contains the items in that group